docs/src/content/docs/for-administrators/configuring-extra-files.md

```diff
@@ -53,6 +53,12 @@ secrets:
 You can also use the `raw` secret type, but be aware that keeping credentials in plain text in your configuration file can be a security hazard.
 :::
 
+## Set S3 policy
```

Comment:
Thanks could you still link the issue with the race-condition here? we don't use this option now but probably good to document the issue here for administrators: #4593